USER-GUIDE.md

```diff
@@ -0,0 +1,362 @@
+# **KCP-Edge** User Guide
+
+
+Table of contents:
+
+- [1. Install **KCP-Edge** pre-requisites](#1-install-kcp-edge-pre-requisites)
+  - [a. `kubectl`](#a-kubectl)
+  - [b. `kcp`](#b-kcp)
+- [2. Install and run **KCP-Edge**](#2-install-and-run-kcp-edge)
+- [3. Create a **KCP-Edge** Inventory Management Workspace (IMW) and Workload Management Workspace (WMW)](#3-create-a-kcp-edge-inventory-management-workspace-imw)
+- [4. Example deployment of nginx workload into a kind local cluster](#4-Example-deployment-of-nginx-workload-into-a-kind-local-cluster)
+  - [a. Stand up a local florin kind cluster](#a-Stand-up-a-local-florin-kind-cluster)
+  - [b. Create a sync target, placement, and edge syncer for onboarding the created florin edge cluster](#b-Create-a-sync-target-placement-and-edge-syncer-for-onboarding-the-created-florin-edge-cluster)
+  - [c. Create the nginx workload and deploy it to the florin cluster](#c-Create-the-nginx-workload-and-deploy-it-to-the-florin-cluster)
+- [5. Cleanup the environment](#5-Cleanup-the-environment)
+
+
+This guide is intended to show how to quickly bring up a **KCP-Edge** environment with its dependencies from a binary release.
+
+## 1. Install **KCP-Edge** pre-requisites
+
+### a. `kubectl`
+
+Detailed installation instructions for different operative systems are available [here](https://kubernetes.io/docs/tasks/tools/).
+
+### b. `kcp`
+
+Since `KCP-Edge` leverages [`kcp`](kcp.io) logical workspace virtualization capability, we have to install `kcp` v0.11.0 binaries first by following the detailed installation instructions available [here](https://docs.kcp.io/kcp/main/) or by using the script below:
+
+```bash
+bash <(curl -s https://raw.githubusercontent.com/kcp-dev/edge-mc/main/hack/install-kcp-with-plugins.sh) --version v0.11.0 --folder $(pwd)/kcp --create-folder
+export PATH="$PATH:$(pwd)/kcp/bin"
+```
+
+Start **kcp** with the following command:
+
+```bash
+kcp start >& kcp_log.txt &
+export KUBECONFIG="$(pwd)/.kcp/admin.kubeconfig"
+```
+
+It should be noted that, when **kcp** is running with the command above, it listens to all of the host's non-loopback addresses and pick one to put in the generated `admin.kubeconfig` file. While this works fine in most cases, such as when using a kind cluster locally (see later), sometimes it may be necessary to specify a public/external ip address that can be reached by remote clusters. For this purpose, the following command should be used:
```

Comment:
s/pick/picks/

Reply:
I will add this detailed instruction with the alternate start command in the newly updated USER_GUIDE.md file